tests/foreman/ui/test_errata.py

```diff
@@ -525,6 +527,49 @@ def test_positive_content_host_search_type(session, erratatype_vm):
         assert errata_ids == sorted(FAKE_9_YUM_SECURITY_ERRATUM)
 
 
+@pytest.mark.skip_if_open("BZ:1655130")
+@tier3
+def test_positive_content_host_errata_details(session, erratatype_vm, module_org, test_name):
+    """ Read for errata details on a content_host by user with only viewer permission
+
+    :id: 236de56f-8035-4072-b0fa-03abbf3fc692
+
+    :Steps:
+
+        1. Content Host with applicable security errata.
+        2. Create a User with 'Viewer' Permission.
+        3. Go to Content Hosts -> Select content host -> Errata Tab -> Select Errata_Id.
+
+    :expectedresults: The errata details apge should be disaplyed for User.
+
+    :BZ: 1655130
+
+    :CaseLevel: Integration
+    """
+    login = gen_string('alpha')
+    password = gen_string('alpha')
+    viewer_role = entities.Role().search(
+        query={'search': u'name="Viewer"'})
+    default_loc_id = entities.Location().search(
+         query={'search': 'name="{}"'.format(DEFAULT_LOC)})[0].id
```

Comment:
Any specific reason to use `DEFAULT_LOC`? Can we not create a new one here?

Reply:
avoid increase time and this is a UI test we are seeing issues where we are selecting the location causing the satellite to take time load. I have seen even other tests most of them have used the default location. 

What are the befits are you seeing adding the different location?

Reply:
Benefits are as similar to create a new org ... but it was a suggestion only ... 
as you mentioned it's already using (which I think not a right way) at mutiple place we can ignore it here and can handle under separate PR ... thanks